### PR TITLE
DAOS-3796 tests: Verify rebuild widely striped

### PR DESCRIPTION
**Pull request by saurabhtandan**
Adding test to verify rebuild with widely striped
objects.

Signed-off-by: Saurabh Tandan <saurabh.tandan@intel.com>

### CODE REVIEW

File: src/tests/ftest/rebuild/rebuild_widely_striped.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        self.pool.wait_for_rebuild(False, interval=1)
+
+        # create 2nd container
+        self.add_container(self.pool)
+        # start 2nd mdtest job
+        thread = threading.Thread(target=self.execute_mdtest)
+        thread.start()
+        time.sleep(3)
+
+        # Kill rank[6] in the middle of mdtest run and
```
Comment (shimizukko) [inline src/tests/ftest/rebuild/rebuild_widely_striped.py:91]:
```suggestion
        # Kill rank[5] in the middle of mdtest run and
```
typo?

File: src/tests/ftest/rebuild/rebuild_widely_striped.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+        # create 3rd container
+        self.add_container(self.pool)
+
+        # start 3rd mdtest job
+        thread = threading.Thread(target=self.execute_mdtest)
+        thread.start()
+        time.sleep(3)
+
+        # Kill 2 server ranks [5,6]
```
Comment (shimizukko) [inline src/tests/ftest/rebuild/rebuild_widely_striped.py:106]:
```suggestion
        # Kill 2 server ranks [3,4]
```
typo?

**Review by shimizukko — APPROVED** [review]
LGTM

**Comment by kprantis** [comment]
Can we please prioritize reviewing this patch? It is one of a couple remaining 1.1.3 tickets which we are trying to close out imminently.

File: src/tests/ftest/util/test_utils_pool.py [hunk 3]
```diff
@@ -481,6 +482,27 @@ def wait_for_rebuild(self, to_start, interval=1):
...
+            self.log.info(
+                "Waiting for pool query to be responsive %s",
+                " with a {} second timeout".format(
+                    self.pool_query_timeout.value))
+
+            end_time = time() + self.pool_query_timeout.value
+            while time() < end_time:
+                try:
+                    self.dmg.pool_query(self.pool.get_uuid_str())
+                    self.log.info("Pool query still responsive")
```
[the remark this answers is not in the document]
Comment (saurabhtandan) [reply src/tests/ftest/util/test_utils_pool.py]: Done

File: src/tests/ftest/rebuild/rebuild_widely_striped.py (hunk 1 above)
Comment (saurabhtandan) [reply src/tests/ftest/rebuild/rebuild_widely_striped.py]: Done

File: src/tests/ftest/rebuild/rebuild_widely_striped.py (hunk 2 above)
Comment (saurabhtandan) [reply src/tests/ftest/rebuild/rebuild_widely_striped.py]: Done

**Review by shimizukko — APPROVED** [review]
LGTM

**Review by sylviachanoiyee — APPROVED** [review]

**Pull request merged**